common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -208,6 +210,9 @@ def find_matching_course_indexes(self, branch=None, search_targets=None):
             for key, value in search_targets.iteritems():
                 query['search_targets.{}'.format(key)] = value
 
+        if org_target:
+            query['org'] = org_target
+
```

Comment:
NOTE: I couldn't use the existing 'search_targets' parameters because that only filters arbitrary keys in a 'search_targets_' sub-dict in the mongo document. 'org' is at the root of the document.